frontend/public/tinymce/langs/zh_TW.js

```diff
@@ -0,0 +1,419 @@
+tinymce.addI18n('zh_TW',{
+"Redo": "\u91cd\u505a",
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 0.](https://app.codacy.com/gh/dataease/dataease/pullRequest?prid=7687731)
```suggestion
  "Redo": "\u91cd\u505a",
```